cpp/oneapi/dal/backend/primitives/objective_function/test/fixture.hpp

```diff
@@ -291,6 +291,18 @@ class logloss_test : public te::float_algo_fixture<Param> {
         }
     }
 
+    float_t clip_prob(float_t prob) {
```

Comment:
Could you add a small comment to explain why you pick `1e-15` and `1e-7` and why we can't use `std::numeric_limits` ?

Reply:
In fact I took these numbers from scikit-learn implementation